solutions/standard/main.tf

```diff
@@ -63,6 +66,7 @@ module "kms" {
 #######################################################################################################################
 
 module "elasticsearch" {
+  count                         = local.use_existing_db_instance ? 0 : 1
```

Comment:
You probably need to add `moved { }` block to avoid the instance being recreated on upgrade

Reply:
Can we hold off adding a moved block until #244 is merged and the PR pipeline run here.

- sometimes terraform magic makes this sort of change just work
  - if it does, I would want to check it manually to verify the logs
  - if it does not, it would be valuable to catch it the new test